locales/de.json

```diff
@@ -2113,7 +2113,8 @@
 			"importing": "importiere",
 			"importSubmission": "Abgabe des Schülers/ der Schülerin importieren",
 			"onTheSubject": "Zum Thema",
-			"save": "Speichern und Abschicken",
+			"saveAndSend": "Speichern und Abschicken",
```

Comment:
Soll hier nicht nur um Speichern gehen und nur "Speichern" stehen?

Reply:
Wird noch im Grading verwendet und nicht mehr in der Submission.